static/src/javascripts/projects/common/modules/onward/more-tags.js

```diff
@@ -9,15 +9,16 @@ define([
     $,
     bean
 ) {
+    var ACTIVE_STATE = 'is-available';
 
     function MoreTags() {
         this.init = function () {
             var $more = $('.js-more-tags');
             if ($more.length !== 0) {
-                $more.removeClass('modern-hidden');
+                $more.addClass(ACTIVE_STATE);
                 bean.on(document.querySelector('.js-more-tags__link'), 'click', function () {
                     $('.modern-hidden-tag').removeClass('modern-hidden');
```

Comment:
One question here, currently this code will show *any* `modern-hidden-tag` elements on the page, rather than just those in the container you acted on. Is there only ever one tag set on the page normally? Guessing so otherwise this would've caused issues by now.